scripts/import/import_variant_synonyms

```diff
@@ -165,6 +167,45 @@ elsif($source_name eq 'UniProt'){
   add_synonyms_uniprot(\%synonym, $source_name, $variation_set_id, $dba, $species); 
 }
 
+elsif($source_name eq 'EVA') {
+  my $result_synonyms = parse_eva_file($data_file);
+
+  # Source EVA
+  my %source_eva = (
+  "EVA" => {
+    description => "Short variant data imported from EVA",
+    url         => "https://www.ebi.ac.uk/eva/",
+    status      => "germline",
+  }, 
+  );
+
+  # get source info 
+  $source_description ||= $source_eva{$source_name}->{description};
+  $source_url ||= $source_eva{$source_name}->{url};
+  my $source_status = $source_eva{$source_name}->{status};
+
+  my $source = get_source($species, $dba, $source_name, $source_version, $source_url, $source_description, $source_status);
+
+  import_synonyms($result_synonyms, $source, $dba, $species);
+}
+
+elsif($source_name eq 'pig_chip') {
+  my $int_dba = $reg->get_DBAdaptor('multi', 'intvar') || warn "Error getting intvar db adaptor\n";;
+
+  # Source Pig SNP Consortium
+  my %source_data = (
+    "name"           => "Pig SNP Consortium",
+    "version"        => undef,
+    "description"    => "PorcineSNP60 BeadChip",
+    "url"            => "https://www.animalgenome.org/repository/pig/",
+    "somatic_status" => "germline"
+  );
+
+  my $source = get_source($species, $dba, $source_data{name}, $source_data{version}, $source_data{url}, $source_data{description}, $source_data{somatic_status});
```

Comment:
Why not allow to override parameters here as well (e.g. description, url, status), like in the case of EVA?